x-pack/plugins/upgrade_assistant/public/application/components/overview/overview.tsx

```diff
@@ -97,7 +107,11 @@ export const Overview: FunctionComponent = () => {
 
         <EuiSteps
           steps={[
-            getBackupStep({ cloud, cloudBackupStatusResponse }),
+            getBackupStep({
+              cloud,
+              isComplete: isStepComplete('backup'),
+              setIsComplete: setCompletedStep.bind(null, 'backup'),
```

Comment:
we probably need to memoize this, otherwise if this is added into a dependency from a useEffect it will be called ∞ 🤔

Reply:
Seems fine now that it's not used as a dependency here: https://github.com/elastic/kibana/pull/111243/files#diff-c79c9d11f20a53d4cae48f26e280ee0ae4c7295aae8a0aebfa189df120f6090cR55

Reply:
If it's memoized, will we we not need to add `// eslint-disable-next-line react-hooks/exhaustive-deps`?